arci-ros/tests/test_robot_client.rs

```diff
@@ -0,0 +1,76 @@
+#[cfg(target_os = "linux")]
```

Comment:
I've merged, but this means that this files' L2 is included.
It cases error in other OSs.
#![cfg] will work for whole file.

Reply:
And it can be run on MacOS not only linux.